rtvi-client-js/src/core.ts

```diff
@@ -313,6 +313,7 @@ export abstract class Client extends (EventEmitter as new () => TypedEmitter<Voi
               this._abortController!
             );
           } else {
+            // @ts-ignore
```

Comment:
I keep seing this error in my IDE:
```
Argument type {mode: "cors", headers: {'Content-Type': string}, method: string, body: string, signal: AbortSignal | null} is not assignable to parameter type RequestInit | undefined 
```

So, I have added it just as a workaround to stop it while I was looking at the other warnings. But I was not supposed to have commited this one. 
I need to figure out why is it complaining, because it seems that everything works just fine.